.github/workflows/tag.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish package to Maven Central Repository
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
```

Comment:
Are we not going to publish when rc tag is pushed?
like v1.0.0-rc2?

Reply:
Oh, Good feedback, I'll add regex for rc*